src/components/accessories/opds/filter/styles.scss

```diff
@@ -57,6 +57,9 @@
             @include susy-media($watch) {
               width: 100%;
             }
+            &.col-4 {
+              width: 100%;
+            }
```

Comment:
This is not the proper way to use col-4 as semantic class.

I suggest to change the selector name in something more "exclusive" and then fix the width that shouldn't be 33%?